jenkins/spark-nightly-build.sh

```diff
@@ -180,15 +163,20 @@ installDistArtifact ${DEFAULT_CUDA_CLASSIFIER}
 distWithReducedPom "install"
 
 if [[ $SKIP_DEPLOY != 'true' ]]; then
-    distWithReducedPom "deploy"
-
     # this deploys selected submodules that is unconditionally built with Spark 3.2.0
     $MVN -B deploy -pl "!${DIST_PL}" \
         -Dbuildver=$SPARK_BASE_SHIM_VERSION \
         -DskipTests \
         -Dmaven.scaladoc.skip -Dmaven.scalastyle.skip=true \
         $MVN_URM_MIRROR -Dmaven.repo.local=$M2DIR \
         -Dcuda.version=$DEFAULT_CUDA_CLASSIFIER
+
+    # try move tmp artifacts back to target folder for simplifying separate release process
+    if (( ${#CLASSIFIERS_ARR[@]} > 1 )); then
+        mv ${TMP_PATH}/${ART_ID}-${ART_VER}-*.jar ${DIST_PATH}/target/
+    fi
+    # Deploy dist jars in the final step to ensure that the POM files are not overwritten
+    jenkins/deploy.sh
```

Comment:
nit: it would be nice to explicitly put some defaults ENV at the place to call the script like SERVER_URL for the new nightly deployment change

Reply:
updated.



Set default environments to deploy artifacts to the internal maven repo